extensions/amp-list/0.1/amp-list.js

```diff
@@ -156,6 +149,17 @@ export class AmpList extends AMP.BaseElement {
     this.ssrTemplateHelper_ = new SsrTemplateHelper(
         TAG, viewer, this.templates_);
 
+    const originTrialPromise = originExperimentsForDoc(this.getAmpDoc())
```

Comment:
discussed offline but we want short-circuit and `promise.resolve` immediately if no `load-more`